docs/utils/README.md

```diff
@@ -4,7 +4,11 @@ The documentation for this library has been generated using [lazydocs](https://g
 
 ## Steps to generate documentation
 
-Run `script.py`:
-1. Module name as prefix: we can leverage this for indexing but then we have to truncate them out. Look into `create_file_index()` in `scripts.py`
-2. Lazydocs built-in ignore-module doesn't work properly with its generator, in this case `ads/__init__.py` generates into `ads.md` which is "accidentally" duplicates with our `ads/ads.py` model. So that we have to use our own `IGNORED_MODULES` to filter out this file.
-3. At the end of the process, to display on https://developers.pinterest.com we append all doc to `skeleton-spec.yaml` results in `python-sdk-doc.yaml`.
+Make sure you are in virtual environment, and run `script.py`:
+```
+python3 docs/utils/scripts.py
+```
+
+## Documentation guide
+
+If you make change to this package, you have to regenerate doc, especially changes that add or remove module, you have to update `MODULES` and `IGNORED_FILES` under `script.py`
```

Comment:
In a future PR, could you add a consolidated version linking to this README in the main README too?

Reply:
Yea, I will make lazydocs generate the overview read me file, then link that file to the main readme